### PR TITLE
[AVA] Update Readme strings and description

### PR DESCRIPTION
**Pull request by giakas**

### CODE REVIEW

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+Azure Video Analyzer provides a platform to build intelligent video applications that span the edge and the cloud. The platform offers the capability to capture, record, and analyze live video along with publishing the results, video and video analytics, to Azure services in the cloud or the edge. It is designed to be an extensible platform, enabling you to connect different video analysis edge modules (such as Cognitive services containers, custom edge modules built by you with open-source machine learning models or custom models trained with your own data) to it and use them to analyze live video without worrying about the complexity of building and running a live video pipeline.
 
-Use the client library for Video Analyzer on IoT Edge to:
+Use the client library for Video Analyzer Edge to:
```
Comment (joshfree) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md:5]: should Edge be removed here as well?

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2021-05-24)
 
+- Updated the title and description of the package and updates to readme.
```
Comment (YijunXieMS) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md:5]: 1. I see 1.0.0-beta.2 is also 2021-05-04. So are we releasing two versions on the same day?
2. Usually change log is put into subsections like [key vault changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md). @g2vinay where is the template for change log?

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+Azure Video Analyzer provides a platform to build intelligent video applications that span the edge and the cloud. The platform offers the capability to capture, record, and analyze live video along with publishing the results, video and video analytics, to Azure services in the cloud or the edge. It is designed to be an extensible platform, enabling you to connect different video analysis edge modules (such as Cognitive services containers, custom edge modules built by you with open-source machine learning models or custom models trained with your own data) to it and use them to analyze live video without worrying about the complexity of building and running a live video pipeline.
```
Comment (joshfree) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md:3]: Should this be Azure Video Analyzer Edge or should it be Azure Video Analyzer?

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md (hunk 1 above)
Comment (giakas) [reply sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md:5]: Library name has Edge, so we are keeping edge here as the name of the library

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md (hunk 3 above)
Comment (giakas) [reply sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md:3]: Azure Video Analyzer is the service name, so we should not have Edge here

**Review by joshfree — APPROVED** [review]
left comment on one spot where Edge was removed from the name.  Just need to make this consistent (one way or another)

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md [hunk 4]
```diff
@@ -166,6 +166,9 @@ MethodResult setPipelineResult = invokeDirectMethodHelper(dClient, setPipelineTo
 
 ## Troubleshooting
 
+When sending a method request using the IoT Hub's `invoke` remember to not type in the method request name directly. Instead use `MethodRequestName.getMethodName()`
```
Comment (YijunXieMS) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md:169]: This doesn't look like other other SDK's troubleshooting, like [app config](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration#troubleshooting).

**Pull request merged**